contexts/pathogen-context/filter-update-steps.tsx

```diff
@@ -1,21 +1,21 @@
 import { MapRef } from "react-map-gl"
-import { ArboAction, ArboStateType } from "./arbo-context"
 import { pipe } from "fp-ts/lib/function.js";
 import { adjustMapPosition } from "./filter-update-steps/adjust-map-position";
 import { updatePediatricAgeGroupFilter } from "./filter-update-steps/update-pediatric-age-group-filter";
 import { addActionToSelectedFilters } from "./filter-update-steps/add-action-to-selected-filters";
 import { applyNewSelectedFilters } from "./filter-update-steps/apply-new-selected-filters";
 import { updateSerotypeFilter } from "./filter-update-steps/update-serotype-filter";
+import { PathogenContextAction, PathogenContextState } from "./pathogen-context";
 
-export interface HandleArboFilterUpdateInput {
-  state: ArboStateType,
-  action: ArboAction,
+export interface HandleFilterUpdateInput {
+  state: PathogenContextState,
+  action: PathogenContextAction,
   map: MapRef | undefined
 }
 
-export type HandleArboFilterUpdateOutput = HandleArboFilterUpdateInput;
+export type HandleFilterUpdateOutput = HandleFilterUpdateInput;
 
-export const handleArboFilterUpdate = (input: HandleArboFilterUpdateInput): HandleArboFilterUpdateOutput => {
+export const handleFilterUpdate = (input: HandleFilterUpdateInput): HandleFilterUpdateOutput => {
```

Comment:
In my perfect world, you would be able to specify whether a step is for arbo or sars cov2 and there would be a list of all the steps and which arboviruses the step applies to in this file.

I'm putting that off for a bit because this PR is big enough as-is but that's something that's coming after this is merged and things are a bit cleaner.